packages/icons/src/icon/Icon.tsx

```diff
@@ -55,28 +40,24 @@ export const Icon = forwardRef<HTMLSpanElement, IconProps>(function Icon(
       };
 
   return (
-    <span
+    <svg
       className={cx(
         withBaseName(),
         { [withBaseName(size as string)]: isNamedSize },
         className
       )}
       style={style}
+      role="img"
+      viewBox="0 0 12 12"
       {...rest}
       ref={ref}
     >
-      <span aria-hidden="true" className={withBaseName("content")}>
-        {isValidElement(children)
-          ? cloneElement(children, {
-              ...SVGProps,
-              className: cx(
-                // eslint-disable-next-line @typescript-eslint/no-unsafe-member-access
-                children.props.className,
-                withBaseName("Svg-content")
-              ),
-            })
-          : children}
-      </span>
-    </span>
+      {isValidElement(children)
+        ? cloneElement(children, {
+            ...children.props,
```

Comment:
I should remove this spread

Reply:
This has affected support for multi-path icons. This should be changed to map through the children.

Reply:
Thanks, updated.

Was thinking another option could be to have `<g aria-hidden>{children}</g>` which would avoid mapping over/cloning children. I don't think this would impact the styling/position of the image. What do you think?